pincer/middleware/interaction_create.py

```diff
@@ -177,7 +177,7 @@ async def interaction_create_middleware(
             else:
```

Comment:
i get a feeling that this condition could be reversed or something to simplify it

Reply:
I have no idea what the hell this does lol. This is gonna take a little bit.

Reply:
I don't even know if this code does anything

Reply:
I see what it does. It just runs the `on_command_error` function when a command raises an exception. I don't think it works very well though since I have no idea what the intended way to use it is.

Reply:
Im not going to touch this in this PR. Its too complicated for this.

Reply:
ok, no problem